### DISCUSSION TITLE
Deno throws when using jose.importJWK and ES256?

### DISCUSSION THREAD

**Discussion by jdjkelly** [panva/jose#333] [Q&A] [answered]
I'm using Deno v1.16.3 and jose 4.3.7. 

This simple example:

```
import * as jose from "https://deno.land/x/jose@v4.3.7/index.ts"

const ecPublicKey = await jose.importJWK({
  crv: 'P-256',
  kty: 'EC',
  x: 'ySK38C1jBdLwDsNWKzzBHqKYEE5Cgv-qjWvorUXk9fw',
  y: '_LeQBw07cf5t57Iavn4j-BqJsAD1dpoz8gokd3sBsOo'
}, 'ES256')

console.log(ecPublicKey)
```

Throws this exception from Deno's implementation of crypto.subtle.importKey:

```
error: Uncaught NotSupportedError: Unrecognized algorithm name
  return crypto.subtle.importKey('jwk', keyData, ...rest)
                       ^
    at normalizeAlgorithm (deno:ext/crypto/00_crypto.js:199:13)
    at SubtleCrypto.importKey (deno:ext/crypto/00_crypto.js:866:35)
    at parse (https://deno.land/x/jose@v4.3.7/runtime/jwk_to_key.ts:149:24)
    at Module.importJWK (https://deno.land/x/jose@v4.3.7/key/import.ts:258:14)
    at file:///Users/joshuakelly/wrk/deno-experiment/demo.ts:3:32
```

I don't *think* I'm doing anything wrong - this is basically the same thing as what's in jose's importJWK example itself.

**Comment by panva** [comment, marked as answer]
As per #266 - ⚠️ Note that Deno's implementation of Web Cryptography API is still [underway](https://github.com/denoland/deno/issues/11690).

`importKey` is not implemented in Deno yet.

> I don't think I'm doing anything wrong

You're not doing anything wrong. Deno's Web Cryptography API just needs to mature.
